src/components/DragDrop/DragDrop.stories.tsx

```diff
@@ -0,0 +1,329 @@
+import { Story, StoryObj, Meta } from '@storybook/react';
+import React, { ComponentProps } from 'react';
+import { DragDrop, Props } from './DragDrop';
+import MediaBlock from '../../upcoming-components/MediaBlock';
+import Table from '../../upcoming-components/Table';
+import Card from '../Card';
+import Heading from '../Heading';
+import TextPassage from '../TextPassage';
+
+export default {
+  title: 'Organisms/Interactive/Drag and Drop',
+  component: DragDrop,
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          margin: '1rem', // Provides spacing around storybook edges
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<Args>;
+
+type Args = ComponentProps<typeof DragDrop>;
+
+const Template: Story<Props> = (args) => <DragDrop {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  items: {
+    'item-1': {
+      title: 'Project #1',
+      children: <div className="fpo">Content here</div>,
+      id: '',
```

Comment:
instead of adding back an empty `id` attribute, could you update `DragDropItem`:
```
 return (
    item.id && (
      <Draggable draggableId={item.id} index={index}>
```
the TS error is because `id` is marked as optional in `item`, but we know that it should already be provided there

Reply:
OR exporting different types for what's passed into `DragDrop` vs. `DragDropItem` (the only difference is whether `id` is optional or not)

Reply:
Thank you, much better.